README.md

```diff
@@ -234,10 +235,10 @@ You have two options, the easiest is to uninstall MySQL and install MariaDB. The
 Alternatively, follow these steps:
 
 1. As root, edit the following file: `/etc/mysql/mysql.conf.d/mysqld.cnf`
-1. Under the line `[mysqld]`, add the following:
+2. Under the line `[mysqld]`, add the following:
```

Comment:
These need to stay as 1's because markdown automatically turns them into an ordered list. If you number them then you have to keep modifying them if things change.

Reply:
I see. I was aware that markdown would indeed turn them into ordered list but did not realized that this is done intentionally. 

The pro of using all 1's instead of increasing numbers:
- we do not need to care about the numbers when new items are inserted, 
  - although maybe the content does not change frequently.

The con of using all 1's instead of increasing numbers:
- it is less readable if someone is reading raw markdown (like me when do proofreading)
  - although most of the people are reading the rendered content

Thanks for the explanation! Let us just keep it as is now.